README.md

```diff
@@ -1822,6 +1822,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Samuel Adu](https://github.com/theadusamuel)
   - [Samuel Kalu](https://github.com/eskayML)
   - [Samuel Z. Bouye](https://github.com/zamblebi)
+  - [Samuel Okello](https://github.com/Okellosam21)
```

Comment:
I believe your name must be one line above (`O` comes first and later on `Z`).
Let us know if you need any help updating that. 🙂